### PR TITLE
Filter explores summary

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #766 #767 

Preview:
<a href="https://www.loom.com/share/56a0af7eb5a9409c89281fea8bfe4164">
    
  </a>

### CODE REVIEW

File: packages/backend/src/services/ProjectService/ProjectService.ts [hunk 1]
```diff
@@ -267,6 +270,44 @@ export class ProjectService {
...
+                    errors: explore.errors,
+                };
+            }
+            return {
+                name: explore.name,
+                tags: explore.tags,
+            };
+        });
+
+        if (filtered) {
```
Comment (owlas) [inline packages/backend/src/services/ProjectService/ProjectService.ts:293]: @ZeRego when are requests not filtered?

Comment (owlas) [reply packages/backend/src/services/ProjectService/ProjectService.ts:293]: I imagine that if I've configured my project to exclude certain explores, I can never get them back. i.e. the filtering happens at compile time, not here

File: packages/frontend/src/hooks/useProjectTablesConfiguration.ts [hunk 2]
```diff
@@ -38,6 +38,7 @@ export const useUpdateProjectTablesConfiguration = (projectUuid: string) => {
         {
             mutationKey: ['tables_configuration_update'],
             onSuccess: async (data) => {
+                await queryClient.invalidateQueries('tables');
```
Comment (owlas) [inline packages/frontend/src/hooks/useProjectTablesConfiguration.ts:41]: Woohooo!

**Review by owlas — COMMENTED** [review]
One question about whether having a filtering option makes sense.

File: packages/backend/src/services/ProjectService/ProjectService.ts (hunk 1 above)
Comment (ZeRego) [reply packages/backend/src/services/ProjectService/ProjectService.ts:293]: For the tables configuration page, I need all the explores for the auto complete inputs.

Comment (owlas) [reply packages/backend/src/services/ProjectService/ProjectService.ts:293]: 👍 didn't think about that. Seems like two different operations:

1. List all models in dbt project
2. List all explores in Lightdash project

But this is simple! Let's do it

**Review by owlas — APPROVED** [review]

**Pull request merged**